### ISSUE TITLE
Missing npm-run-all

### ISSUE THREAD

**Issue by jamesjjk** [FGRibreau/mailchecker#130]
Latest build  mailchecker@3.0.37 has an error. See below

```
Compiling libraries from templates...
[ loaded:list ]	 5228
[ compiling:template ]	 mailchecker.tmpl.clj
[ compiling:template ]	 mail_checker.tmpl.ex
[ compiling:template ]	 MailChecker.tmpl.js
[ compiling:template ]	 index.tmpl.js
[ compiling:template ]	 MailChecker.tmpl.php
[ writing:template ]	 mail_checker.tmpl.ex
[ writing:template ]	 mailchecker.tmpl.clj
[ writing:template ]	 index.tmpl.js
[ compiling:template ]	 MailChecker.tmpl.py
[ compiling:template ]	 mail_checker.tmpl.rb
[ writing:template ]	 MailChecker.tmpl.php
[ writing:template ]	 MailChecker.tmpl.js
[ compiling:template ]	 lib.tmpl.rs
[ writing:template ]	 mail_checker.tmpl.rb
[ writing:template ]	 MailChecker.tmpl.py
[ writing:template ]	 lib.tmpl.rs
Done.
sh: npm-run-all: command not found

```

**Comment by phongle79** [comment]
@jamesjjk 
You need to install npm-run-all package.

**Comment by Tornquist** [comment]
`npm-run-all` needs to be listed as a dependency. You shouldn't have to manually add dependencies outside of a package's package.json

**Comment by colinhemphill** [comment]
+1. Ran into this on update to 3.0.37. I can start a PR if necessary, but this should be added asap.

**Comment by gierschv** [comment]
+1, sad that the patch release breaks the install :/

**Comment by eguyot** [comment]
+1

**Comment by FGRibreau** [comment]
Deprecated v3.0.37, fixed in v3.0.39
